Libraries/LibWeb/Painting/ImagePaintable.cpp

```diff
@@ -106,7 +115,7 @@ void ImagePaintable::paint(PaintContext& context, PaintPhase phase) const
                 }
                 break;
             case CSS::ObjectFit::ScaleDown:
-                // FIXME: Implement
+                VERIFY_NOT_REACHED(); // handled outside the switch-case
```

Comment:
Can we do a compile-time check here?

Reply:
What would I use for that?

Reply:
Good point. I expected `static_assert(false, "Handled outside the switch-case");` to work, but for some reason it doesn't...
That worked in https://github.com/LadybirdBrowser/ladybird/pull/2734 and I expected it to work here

Reply:
The entire if-else chain in the PR you linked is `constexpr` which makes that a much simpler case, I guess.